### ISSUE TITLE
pass/fail radiobuttons in assessment don't show selected state (windows high contrast + edge)

### ISSUE THREAD

**Issue by karanbirsingh** [microsoft/accessibility-insights-web#3435]
## Describe the bug

In Edge + Windows HC, the radio buttons in assessments are always white even if selected. It's therefore unclear what the state is. This issue only shows up when Windows high contrast mode is on. Toggling high-contrast mode from the extension itself (in both chrome and edge) produces the right behavior.

**To Reproduce**

1. Add Accessibility Insights for Web to Edge
2. Turn on Windows High Contrast mode
3. Open an assessment whose results have pass/fail radio buttons (e.g. links)
4. Notice the radio buttons don't show their state

## Screenshots

![screenshot of radio buttons in assessment](https://user-images.githubusercontent.com/7775527/95116505-fdd3b480-06fb-11eb-9718-b6837483a30b.png)

## Context (please complete the following information)

- OS Name & Version: Windows OS 19041.508
- AI-Web Version & Environment: 2.21.1 Production
- Browser Version: Microsoft Edge 86.0.622.31 (Official build) beta (64-bit)

**Comment by dbjorge** [comment]
Potentially related fluentui issues/PRs:

* most similar issue, wontfixed: https://github.com/microsoft/fluentui/issues/13768
* similar issue, fixed in office-ui-fabric-react@v7.121.7: https://github.com/microsoft/fluentui/issues/13767
* PR that fixed 13767: https://github.com/microsoft/fluentui/pull/13846

**Comment by dbjorge** [comment]
In the latest Fluent UI (7.144.2 as of writing), this issue is sort-of resolved; the choicegroup buttons do use our override colors in that version, but we're providing override colors that don't meet contrast requirements, so it's a different problem rather than a complete fix:

![image](https://user-images.githubusercontent.com/376284/95121777-8747ae80-071d-11eb-8ccb-f00248b09845.png)

(there are also a variety of other, unrelated high-contrast bugs introduced by upgrading from our current fabric version to fluentUI 7.144.2)

**Comment by ferBonnin** [comment]
@iamrafan can we please take a look at the fluent UI update to see if it introduces other HC issues? @dbjorge was this pushed to playground?

**Comment by dbjorge** [comment]
@ferBonnin Just pushed it to [playground](https://chrome.google.com/webstore/detail/accessibility-insights-fo/oiokpkjanjfndgbdgepalohbolalekek) now (sorry for the delay, needed to fix a few unit tests before it became releasable), should be there as soon as it clears store review.

**Comment by dbjorge** [comment]
I spent an hour or two investigating more immediate workarounds. I think the "right" workaround would probably be along the lines of:

* Add a detector to the background page similar to our unified HC detector that is along the lines of on `window.matchMedia('(forced-colors: active)').addListener(mq => userConfigActionCreator.nativeHighContrastTransition(mq.matches))`;
* Use that to set our fabric + local HC styles in system mode
* use `forced-color-adjust: none` on the button colors

**Comment by lisli1** [comment]
In Edge Version 87.0.664.66 (Official build) (64-bit) with (1) Windows high contrast on and (2) the extension high contrast both on and off, the checkbox to turn on the visualization for a specific instance is not visible (first image) unless I am clicking on the instance (second image).
![image](https://user-images.githubusercontent.com/48259897/103718529-ccc58300-4f7b-11eb-9a0f-a37f331ff700.png)
![image](https://user-images.githubusercontent.com/48259897/103718578-e797f780-4f7b-11eb-9716-7a889a0db766.png)

**Comment by ferBonnin** [comment]
@ferBonnin to create a feature for OF update.
@iamrafan how did the investigation on playground with the OF update went?

**Comment by iamrafan** [comment]
@ferBonnin I will share the investigation document via email

**Comment by iamrafan** [comment]
@ferBonnin, @karanbirsingh  I validated that this specific issue has been fixed. Now, I see other issues like the undo button not being visible. I think we close this issue and open new ones related to the other bugs that I will share offline.

Validation details:
```AI Web Version 2.27.0```
```Microsoft Edge Version 92.0.902.73```
```Windows 10 Enterprise Version 21H1 OS build 19043.1165```

![image](https://user-images.githubusercontent.com/22068434/129417548-4a1aa174-9279-4ae6-ac26-0f2f6bd29291.png)

**Comment by ferBonnin** [comment]
@jlperkins can you help us verify if this issue still happens after fluent update?

**Comment by jlperkins** [comment]
In Windows 11 HC modes, this issue appears to be resolved. The left-hand checkboxes, radio buttons, and undo buttons all appear with enough contrast, in each of the Windows 11 HC themes. See pictures below. I am unable to verify this for Windows 10 HC modes as I have already upgraded my work computer to Windows 11; a dev with Windows 10 still running should try and verify this issue as well. 

Images of the Link assessment, showing the checkboxes, radio buttons, and undo buttons working: 
![image](https://user-images.githubusercontent.com/33770444/165602519-e3d3afa6-6c09-40e6-9d33-accb9cbb682a.png)
[Theme 1, aquatic]

![image](https://user-images.githubusercontent.com/33770444/165602542-f0018b73-4860-4552-ac6c-c3456bc31254.png)
[Theme 2, desert]

![image](https://user-images.githubusercontent.com/33770444/165602555-6e316dd4-05b5-4097-831f-89d8a599026a.png)
[Theme 3, dusk]

![image](https://user-images.githubusercontent.com/33770444/165602601-aa7219ce-cc21-45dd-8bea-c39a403c61ba.png)
[Theme 4, night sky]
